prime-angular-frontend/src/app/modules/paper-enrolment/shared/services/paper-enrolment-resource.service.ts

```diff
@@ -190,8 +192,12 @@ export class PaperEnrolmentResource {
       );
   }
 
-  public updateAdjudicationDocuments(enrolleeId: number, documentGuids: string[]): Observable<any> {
-    return this.apiResource.put<NoContent>(`enrollees/${enrolleeId}/paper-submissions/documents`, documentGuids)
+
+  public updateAdjudicationDocuments(enrolleeId: number, documentsGuidAndType: {}[]): Observable<any> {
+    const options = {
+      'Content-Type': 'application/json',
+    };
```

Comment:
Does the backend explicitly expects the Content-Type to be set?

Reply:
That actually may not bee needed anymore, was using it to try something, will remove

Reply:
Deleted.